apps/src/templates/SchoolDataInputs.jsx

```diff
@@ -1,99 +1,138 @@
+  const [zip, setZip] = useState('');
+  const [country, setCountry] = useState('');
+  const [schoolName, setSchoolName] = useState('');
+  const [zipSearchReady, setZipSearchReady] = useState(false);
+
+  let COUNTRY_ITEMS = [{value: 'selectCountry', text: i18n.selectCountry()}];
   for (const item of Object.values(COUNTRIES)) {
     COUNTRY_ITEMS.push({value: item.label, text: item.value});
   }
 
+  useEffect(() => {
+    if (zip.length === 5) {
+      setZipSearchReady(true);
+    } else {
+      // Removes the school dropdown if you delete part of the zip
+      setZipSearchReady(false);
+    }
+  }, [zip.length]);
+
+  const onCountryChange = e => {
+    const country = e.target.value;
+    setCountry(country);
+    // We don't want to display any fields to start that won't eventually be
+    // necessary, so updating both of these any time country changes
+    if (country === 'US') {
+      setAskForZip(true);
+      setIsOutsideUS(false);
+    } else {
+      setAskForZip(false);
+      setIsOutsideUS(true);
+    }
+  };
+
   return (
     <div className={style.outerContainer}>
-      <Heading2 className={style.topPadding}>{i18n.censusHeading()}</Heading2>
-      <BodyTwoText>{i18n.schoolInfoInterstitialTitle()}</BodyTwoText>
+      {includeHeaders && (
+        <div>
+          <Heading2 className={style.topPadding}>
+            {i18n.censusHeading()}
+          </Heading2>
+          <BodyTwoText>{i18n.schoolInfoInterstitialTitle()}</BodyTwoText>
+        </div>
+      )}
       <div className={style.inputContainer}>
         <BodyTwoText className={style.padding} visualAppearance={'heading-xs'}>
           {i18n.whatCountry()}
         </BodyTwoText>
         <SimpleDropdown
           className={style.dropdown}
+          name={fieldNames.country}
           items={COUNTRY_ITEMS}
-          name="countryDropdown"
           selectedValue={country}
           onChange={onCountryChange}
           size="m"
         />
-        <BodyTwoText className={style.padding} visualAppearance={'heading-xs'}>
-          {i18n.enterYourSchoolZip()}
-        </BodyTwoText>
-        <input
-          type="text"
-          placeholder={'i.e. 98104'}
-          onChange={() => {}}
-          value={''}
-        />
-        <BodyTwoText className={style.padding} visualAppearance={'heading-xs'}>
-          {i18n.selectYourSchool()}
-        </BodyTwoText>
-        <SimpleDropdown
-          className={style.dropdown}
-          items={TEST_SCHOOL_DATA}
-          name="schoolDropdown"
-          selectedValue={'test'}
-          onChange={() => {}}
-          size="m"
-        />
+        {askForZip && (
+          <label>
+            <BodyTwoText
+              className={style.padding}
+              visualAppearance={'heading-xs'}
+            >
+              {i18n.enterYourSchoolZip()}
+            </BodyTwoText>
+            <input
+              type="text"
+              name={fieldNames.schoolZip}
+              placeholder={'i.e. 98104'}
```

Comment:
genuinely don't know the answer to this but should this be translateable?

Reply:
(Also happy to remove this entirely. I subconsciously added it in but realize it isn't in the figma)

Reply:
I vote for translating if it stays in -- not all US users use our site in English (though I believe the vast majority do). I don't have an opinion on whether to keep it or not though.

Reply:
After doing some [reading about placeholders](https://www.smashingmagazine.com/2018/06/placeholder-attribute/), I'm removing it! A highlight: the contrast is too low to be accessible. A11y guidelines state that the <label> should be sufficient, and if more info is needed, it should live below the label but above the input so it doesn't disappear.